src/foam/core/AbstractArrayPropertyInfo.java

```diff
@@ -6,17 +6,14 @@
 
 package foam.core;
 
-import foam.dao.pg.IndexedPreparedStatement;
+import foam.dao.jdbc.IndexedPreparedStatement;
```

Comment:
Why is this import needed?

Reply:
Not needed, I made a refactoring of packages

Reply:
It is used by the setStatementValue method, I only made a refactoring of the packages